app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -386,17 +384,6 @@ protected void onActivityResult(int requestCode, int resultCode, Intent intent)
                     return;
                 }
                 break;
-            case UPGRADE_APP:
-                if (resultCode == RESULT_CANCELED) {
-                    return;
-                } else if(resultCode == RESULT_OK) {
```

Comment:
Worth noting that this has effectively already been removed because KEY_REQUIRE_REFRESH is no longer being sent, and there don't seem to be any problems coming up with updates, but worth understanding why it's ok.